python/nav/etc/ipdevpoll.conf

```diff
@@ -6,6 +6,10 @@
 #
 logfile = ipdevpoll.log
 
+[netbox_filters]
```

Comment:
This section kinda breaks the following `max_concurrent_jobs` setting, which still belongs in the `[ipdevpoll]` section, not in the `[netbox_filters]` section.